From 6e7004c9e56d4fa509fe1324e7d32ae6d5a5d1e7 Mon Sep 17 00:00:00 2001
From: ricardomorales <ric.morales22@gmail.com>
Date: Sun, 15 Sep 2013 12:04:17 -0400
Subject: [PATCH] Interns Styles

Added style for the interns page, as well as the individual interns
pages.
---
 state/css/global.css                     | 39 ++++++++++++++++++++++++++++++++
 state/css/state-state-default-normal.css |  9 ++++++++
 state/css/state-state-default.css        |  9 ++++++++
 3 files changed, 57 insertions(+)

diff --git a/state/css/global.css b/state/css/global.css
index 75e589a..9eefb99 100644
--- a/state/css/global.css
+++ b/state/css/global.css
@@ -1989,4 +1989,43 @@ div.profile div.field-display-name {
 
 .block-contributor-bio-block-1 p {
   display: inline;
+}
+
+/**
+ * Interns page
+ */
+
+
+.view-interns .view-content .views-row .field-alma-mater {
+  margin: .1em 0 .2em 0;
+}
+
+.view-interns .view-content .views-row .statistics_counter {
+  display: none;
+}
+
+.field-portrait {
+  margin-bottom: .3em;
+}
+
+@media all and (max-width: 740px) {
+  
+.view-interns .view-content .views-row .node-links {
+  float: right;
+}
+
+}
+
+/**
+ * Individual Intern page
+ */
+
+@media all and (max-width: 740px) {
+
+.node-internship .field-portrait img {
+  width: 100%;
+  max-width: 100%;
+  height: auto;
+}
+
 }
\ No newline at end of file
diff --git a/state/css/state-state-default-normal.css b/state/css/state-state-default-normal.css
index a6f8827..31eb2e9 100644
--- a/state/css/state-state-default-normal.css
+++ b/state/css/state-state-default-normal.css
@@ -463,4 +463,13 @@ body.content-top .zone-content,
 .no-sidebars.page-search .region-content-inner,
 .not-front.no-sidebars.node-type-page .region-content-inner {
   width: 80%;
+}
+
+/**
+ * Interns page
+ */
+ 
+.view-interns .view-content .views-row {
+  margin-right: .5em;
+  height: 435px;
 }
\ No newline at end of file
diff --git a/state/css/state-state-default.css b/state/css/state-state-default.css
index 4dcc91a..62c13ab 100644
--- a/state/css/state-state-default.css
+++ b/state/css/state-state-default.css
@@ -1361,4 +1361,13 @@ h1.title,
 .youtube-pull-item:nth-child(2n+2),
 .flickr .content a:nth-child(2n+2) {
   margin-right: 0;
+}
+
+/**
+ * Interns page
+ */
+ 
+.view-interns .view-content .views-row {
+  margin-right: .5em;
+  height: 375px;
 }
\ No newline at end of file
-- 
1.8.3.msysgit.0

